docs/contributing/drivers.rst

```diff
@@ -0,0 +1,30 @@
+.. _contributing-drivers:
+
+Community Drivers
+=================
+
+In addition to the core drivers, maintained by the :ref:`contributing-core`,
+the community can always provide additional drivers. We welcome any submission,
+and new drivers will be hosted under the
+`napalm-automation-community <https://github.com/napalm-automation-community>`_ on
+GitHub. They can be seen as *plugins* for the base NAPALM framework, which is
+why each driver is maintained under its own repository.
+
+Once a new driver is added under the named organisation, the user proposing it
+takes the entire responsibility to maintain. That includes:
+
+- Documentation
+- Fix bugs
+- Triage and manage issues
+- Review and merge Pull Requests
+- API compatibility with the core drivers
+
+While everything is a best-effort of the volunteer, we highly encourage the
+community to avoid submitting drivers they are unable to maintain.
+
+The core developers will provide you the tools and the methodologies to
+create a new driver, together with the associated documentation.
+
+If there is high demand for a specific driver, and its maintainer proves
+and they are able to commit to continous work on the long run, the driver can
```

Comment:
~and~ they are able to commit to *a continuous* ... the driver and the contributor can be promoted to the core group.